src/main/java/org/generationcp/commons/service/impl/BreedingViewImportServiceImpl.java

```diff
@@ -621,7 +622,7 @@ public void importSummaryStatsData(final File file, final int studyId) throws Br
 		final TrialEnvironments trialEnvironments = this.studyDataManager.getTrialEnvironmentsInDataset(datasetId);
 
 		final boolean isSelectedEnvironmentFactorALocation = this.studyDataManager.isLocationIdVariable(studyId, environmentFactorName);
-		final Map<String, String> locationNameToIdMap = this.studyDataManager.createInstanceLocationIdToNameMapFromStudy(studyId).inverse();
```

Comment:
Let's change the variable to locationIdNameMap since the location id is the map key (inverse not done/couldn't be done anymore)